Subsonicious/Subsonicious/Info.plist

```diff
@@ -70,5 +70,10 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+    <key>NSAppTransportSecurity</key>
+    <dict>
+        <key>NSAllowsArbitraryLoads</key>
```

Comment:
Actually the server from where the music come from could be on both http & https. As I don't already know which url are going to be used, I haven't find any better idea :/ 
Do you have a better suggestion apart from restricting users to only https?

Reply:
I don't know, but I don't really have the choice.. From my memories, it is forbidden to use this parameter as-is for apps in the store..

Reply:
Correct, but for now I don't want to restrict the users to only use https servers (I may change my mind later on).
As the doc says
> You must supply a justification during App Store review if you set the key’s value to YES, as described in Provide Justification for Exceptions. Use this key with caution because it significantly reduces the security of your app. In most cases, it’s better to upgrade your servers to meet the requirements imposed by ATS, or at least to use a narrower exception.

https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsallowsarbitraryloads